docs/requirements.txt

```diff
@@ -1,2 +1,2 @@
-sphinx>=3.0
+sphinx>=4.0.dev0
```

Comment:
We should pin docutils==0.17 as well to test

Reply:
Seems you already done it.